[2022-05-03 18:07:31,172] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.task_3 2022-05-03T18:07:20.016868+00:00 [queued]>
[2022-05-03 18:07:31,202] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.task_3 2022-05-03T18:07:20.016868+00:00 [queued]>
[2022-05-03 18:07:31,202] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-05-03 18:07:31,203] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-05-03 18:07:31,203] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-05-03 18:07:31,215] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_3> on 2022-05-03T18:07:20.016868+00:00
[2022-05-03 18:07:31,218] {standard_task_runner.py:52} INFO - Started process 89192 to run task
[2022-05-03 18:07:31,225] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag.processing_tasks', 'task_3', '2022-05-03T18:07:20.016868+00:00', '--job-id', '22', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpqo7bf6d8', '--error-file', '/tmp/tmp82jzxda9']
[2022-05-03 18:07:31,227] {standard_task_runner.py:77} INFO - Job 22: Subtask task_3
[2022-05-03 18:07:31,325] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.processing_tasks.task_3 2022-05-03T18:07:20.016868+00:00 [running]> on host airflowvm
[2022-05-03 18:07:31,479] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag.processing_tasks
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2022-05-03T18:07:20.016868+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-03T18:07:20.016868+00:00
[2022-05-03 18:07:31,479] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-05-03 18:07:31,480] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-05-03 18:07:31,487] {subprocess.py:75} INFO - Output:
[2022-05-03 18:07:34,496] {subprocess.py:83} INFO - Command exited with return code 0
[2022-05-03 18:07:34,556] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag.processing_tasks, task_id=task_3, execution_date=20220503T180720, start_date=20220503T180731, end_date=20220503T180734
[2022-05-03 18:07:34,599] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-03 18:07:34,608] {local_task_job.py:151} INFO - Task exited with return code 0
